examples/create_sacc_clusters.py

```diff
@@ -0,0 +1,108 @@
+import sacc
```

Comment:
I don't want to go into copy-pasta bonanza -- can you instead remove `create_sacc_clusters.py` and just add the similar example functionality to `create_sacc.py` instead? It is all just an example, so I'd rather have one comprehensive example.

Reply:
The top line should follow the other `sacc` examples and be executable in the same way. Put `#!/usr/bin/env python` at the top so that it can be called just like the other examples.

Reply:
I've done this integration.